src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -155,10 +155,334 @@ public SyntaxNode MethodDeclaration(IMethodSymbol method, IEnumerable<SyntaxNode
         }
 
         /// <summary>
-        /// Creates a method declaration.
+        /// Creates an implicit conversion declaration.
+        /// </summary>
+        public virtual SyntaxNode ImplicitConversionDeclaration(
+            IEnumerable<SyntaxNode> parameters = null,
+            SyntaxNode returnType = null,
+            Accessibility accessibility = Accessibility.NotApplicable,
+            DeclarationModifiers modifiers = default(DeclarationModifiers),
+            IEnumerable<SyntaxNode> statements = null)
+        {
+            throw new NotImplementedException();
```

Comment:
This is necessary sometimes. There is precedent in the framework for NotImplementedException and NotSupportedException. I believe NotSupportedException is better as NotImplementedException is often an indication of unfinished code in my experience.

It is a good practice to accompany the unsupported-unless-overridden virtuals with a capability API.